src/main/resources/Documentation/about.md

```diff
@@ -58,12 +58,19 @@ Example:
 
     [plugin "eiffel-integration"]
       enabled = true
-      filter = (release-).* master
+      filter = (release-).*,master
       flow-context = aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0
       remrem-publish-url = https://localhost:8080/publish
       remrem-username = dummyuser
       remrem-password = ********
 
+<span style="color:red">Note</span>
+-----------------
+Manually editing the Eiffel plugin configurations will show the configurations in the GUI using , seperated values.
```

Comment:
s/seperated/separated/

I also think it's more clear and less ambiguous to say "comma-separated" than ", separated".

Reply:
Fixed in https://github.com/eiffel-community/eiffel-gerrit-plugin/pull/35/commits/cf576100c7b69d9081c4e4b8804c060ad7feeff9

Reply:
@magnusbaeck  any other comments, Otherwise I will merge the changes.

Reply:
No, go ahead and merge.